### ISSUE TITLE
Error using `pagecrypt` with Node 19

### ISSUE THREAD

**Issue by metonym** [Greenheart/pagecrypt#34]
Ref: https://github.com/render-examples/pagecrypt/issues/1

[Node version 19](https://nodejs.org/en/blog/announcements/v19-release-announce/) was released last week, and there appears to be a difference in how the `loadCrypto` function is evaluated.

In Node 16, for example, this condition is [not evaluated](https://github.com/Greenheart/pagecrypt/blob/c1590740f228f25c5a56be1d7754943da7ebf236/src/crypto.ts#L7-L10) and `crypto` is loaded dynamically. However, in Node 19, it appears that the condition is evaluated, producing the `ReferenceError: window is not defined` message. I'm unsure if this is a bug in Node 19 or if it's an intentional breaking change.

This is the output when running the CLI using Node 19:

```sh
file:///Users/pagecrypt/cli.js:16
const crypto2 = window.crypto || globalThis.crypto;
                    ^
ReferenceError: window is not defined
```

**Comment by Greenheart** [comment]
@metonym This fix is now available in pagecrypt 6.0.0 :)

**Comment by metonym** [comment]
Awesome, thanks!
